### DISCUSSION TITLE
Support for multiple audio tracks

### DISCUSSION THREAD

**Discussion by NotShocked** [Purfview/whisper-standalone-win#185] [Ideas]
I have some video files that have multiple audio tracks.

Example discord is audio track 1 and my microphone is audio track 2.

Is it possible to tell it which audio track to transcribe?
Or can I have it do all audio tracks?

**Comment by Purfview** [comment]
> Is it possible to tell it which audio track to transcribe?

No.

> Or can I have it do all audio tracks?

No. It works only on the first audio track.

**Comment by Purfview** [comment]
Implemented with `--ff_track`
